build-tools/scripts/RunTests.targets

```diff
@@ -112,19 +112,27 @@
         ContinueOnError="ErrorAndContinue"
         Projects="$(_TopDir)\tests\RunApkTests.targets"
     />
+    <PropertyGroup>
+      <_HostOS>$(HostOS)</_HostOS>
+      <_ExeExtension></_ExeExtension>
+      <_HostOS Condition="'$(_HostOS)' == 'Windows'"></_HostOS>
+      <_ExeExtension Condition="'$(_HostOS)' == 'Windows'">.exe</_ExeExtension>
+      <_CrossCompilerAvailable Condition="Exists('$(_TopDir)\bin\$(Configuration)\lib\xamarin-android\xbuild\Xamarin\Android\$(_HostOS)\cross-arm$(_ExeExtension)')">False</_CrossCompilerAvailable>
```

Comment:
Shouldn't this line be True, not False?  Otherwise, `$(_CrossCompilerAvailable)` will *never* be True...  which will disable e.g. line 130.